### PR TITLE
Databox #3604

### PR DESCRIPTION
**Pull request by lcaresia**
I didn't find a way to do `Increment Count`, so I removed from issue description.

<a href="https://gitpod.io/#https://github.com/PipedreamHQ/pipedream/pull/4753"></a>

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

File: components/databox/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@pipedream/databox",
+  "version": "0.0.2",
```
Comment (jcortes) [inline components/databox/package.json:3]: Shoulnd't you start with?
```suggestion
  "version": "0.0.1",
```

File: components/databox/actions/send-custom-data/send-custom-data.mjs [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import databox from "../../databox.app.mjs";
+
+export default {
+  name: "Send Custom Data",
+  version: "0.0.1",
+  key: "databox-send-custom-adta",
```
Comment (jcortes) [inline components/databox/actions/send-custom-data/send-custom-data.mjs:6]:
```suggestion
  key: "databox-send-custom-data",
```

**Review by jcortes — COMMENTED** [review]
Hi @lcaresia About `Increment Count` action what I understand you need to do is to get the metric `value` by using `getMetrics` once you select the metric you can increment it by one with `sendCustomData`. I've also added a couple of request changes. Thanks!

**Comment by lcaresia** [comment]
@jcortes `getMetrics` only returns metric key and name, not a value!

**Review by jcortes — COMMENTED** [review]
@luancazarine has tested the `Increment Count` use case using postman according to library code and it didn't work either so we decided to move on. Ready for QA!

File: components/databox/actions/send-custom-data/send-custom-data.mjs [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import databox from "../../databox.app.mjs";
+
+export default {
+  name: "Send Custom Data",
+  version: "0.0.9",
```
Comment (michelle0927) [inline components/databox/actions/send-custom-data/send-custom-data.mjs:5]: Is this version correct, or should it be "0.0.1"?

**Review by michelle0927 — APPROVED** [review]

**Pull request merged**